omod/src/main/web/dashboardwidgets/obsacrossencounters/obsacrossencounters.controller.js

```diff
@@ -29,177 +28,107 @@ export default class ObsAcrossEncountersController {
   }
 
-  getConceptWithShortName(concept) {
-    let lastShortName = null;
-    for(let i=0; i < concept.names.length; i++) {
-      let conceptName = concept.names[i];
-      if (conceptName.conceptNameType == 'SHORT') {
-        lastShortName = conceptName.display;
-        if (this.sessionLocale !== null && this.sessionLocale === conceptName.locale) {
-          // we found a SHORT name that matches the locale of the current session
-          break;
-        }
-      }
-    }
-    if (lastShortName !== null && lastShortName.length > 0) {
-      concept.display = lastShortName;
+  getObsValue(obs) {
+    if (['8d4a505e-c2cc-11de-8d13-0010c6dffd0f',
+        '8d4a591e-c2cc-11de-8d13-0010c6dffd0f',
+        '8d4a5af4-c2cc-11de-8d13-0010c6dffd0f'].includes(obs.concept.datatype.uuid)) {
+        // If value is date, time or datetime
+        return this.$filter('date')(new Date(obs.value), this.config.dateFormat);
+    } else if (angular.isDefined(obs.value.display)) {
+        // If value is a concept
+        return this.config.useConceptNameForValue ? obs.value.concept.display : obs.value.display
+    } else {
+        return obs.value;
     }
-    return concept;
   }
 
-
-  parseEncounters(encounters) {
-    angular.forEach(encounters, (encounter) => {
-      let searchObs = {};
-      angular.forEach(encounter.obs, (obs) => {
-        let conceptKeys = Object.keys(this.conceptsMap);
-
-        if (conceptKeys.findIndex(conceptKey => conceptKey === obs.concept.uuid) >= 0) {
-          //we found an obs match
-          searchObs[obs.concept.uuid] = obs;
+  addToSimpleEncs(encounters) {
+    for (let encounter of encounters) {
+      const conceptKeys = Object.keys(this.conceptsMap);
+      // all normal concepts will go in one row
+      const foundObsByUuid = {};
+      for (let obs of encounter.obs) {
+        // see if the concept matches
+        if (conceptKeys.includes(obs.concept.uuid)) {
+          foundObsByUuid[obs.concept.uuid] = obs;
         }
-        if (obs.groupMembers != null && obs.groupMembers.length > 0) {
-          // need to search the groupMembers
-          let foundObs = this.parseGroupMembers(obs.groupMembers, conceptKeys);
-          if (typeof foundObs !== 'undefined' && foundObs !== null && Object.keys(foundObs).length > 0) {
-            if (Object.keys(foundObs).every(item => conceptKeys.includes(item))) {
-              //this is a complete pair of matching obs with a given concept uuid within the same obsGroup
-              let enc = {
-                encounterType: encounter.encounterType.name,
-                encounterDatetime: encounter.encounterDatetime,
-                obs: foundObs
-              };
-              this.simpleEncs.push(enc);
-            } else {
-              // we have an incomplete match, just add the values to the existing object
-              for (key in Object.keys(foundObs)){
-                searchObs[key] = searchObs[key];
-              }
```

Comment:
I don't *think* there's a functional change here. I'm pretty sure this code was unreachable, and it definitely does nothing. But it's hard to say how the function used to work, altogether.